Automattic/pocket-casts-ios
# [Kids Profile] Add Tracks events

| 📘 Part of: #1935
|:---:|

Fixes #1940 

Tracks events added to the Kids Profile

## To test

> [!NOTE]
> Make sure `kidsProfile` feature is enabled

• Run the app and go to your profile
• You should see the Kids profile banner (if not check the FF panel or reset your user defaults)
• You shuld see `🔵 Tracked: kids_profile_banner_seen`
• Tap on `Request Early Access`
• You should see `🔵 Tracked: kids_profile_early_access_requested`
• You should see `🔵 Tracked: kids_profile_thank_you_for_your_interest_seen`
• If you tap on NO, you should see `🔵 Tracked: kids_profile_no_thank_you_tapped`
• If you tap on Send Feedback, You should see `🔵 Tracked: kids_profile_send_feedback_tapped`
• The text editor should appear and you should see `🔵 Tracked: kids_profile_feedback_form_seen`
• If you tap on Send you should see `🔵 Tracked: kids_profile_feedback_sent`
• If you tap on the `X` in the top right of the banner to dismiss it, you should see `🔵 Tracked: kids_profile_banner_dismissed`

## Checklist
